webrecorder/browsertrix
# Refactor collections and browser profile data-tables

Resolves https://github.com/webrecorder/browsertrix-cloud/issues/1483

@Shrinks99 while we're still settling on data table variations, I added a shadow to collection and browser profile items to match workflows more for now.



### Changes

- Updates browser profile list styles to match other data table styles
- Makes entire collection item clickable
- Refactors row click area to fix text overflow

### Manual testing

1. Log in and go to browser profiles. Verify items can be clicked to view details and menu items work
2. Go to Collections. Verify items can be clicked to view details and menu items work

### Screenshots

| Page | Image/video |
| ---- | ----------- |
| Browser Profiles | <img width="1312" alt="Screenshot 2024-01-30 at 5 05 21 PM" src="https://github.com/webrecorder/browsertrix-cloud/assets/4672952/6e662575-fa19-460a-80d0-5f8a87d64eca"> |
| Collections | <img width="1330" alt="Screenshot 2024-01-30 at 4 57 03 PM" src="https://github.com/webrecorder/browsertrix-cloud/assets/4672952/757f092f-3059-4435-a203-e25c1fbf7f56"> |
